packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Share a sign-in session with native mobile apps
+meta:
+  - name: description
+    content: Mobile apps use an external user-agent (the device's browser) to perform authentication. Get information on how you can apply sign-in principles that are similar to web apps to share a sign-in session between mobile apps on a device.
+layout: Guides
+---
+
+This guide uses sample apps to demonstrate how to share a Single Sign-On (SSO) session on a mobile device. After you complete this guide, you should have a better understanding of how a sign-in session is shared between two mobile apps on a device and will be able to use the steps in the guide to help you configure your own apps.
+
+---
+
+**Learning outcomes**
+
+* Persist a session between multiple OIDC mobile apps
+* Clear the session when appropriate
+
+**What you need**
+
+* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup)
+* Android Studio with an emulator for Android testing
+* Xcode with a simulator for iOS testing
+
+**Sample code**
+
+See the [Sample code](#sample-code) section.
+
+---
+
+## Overview
+
+In OAuth, the authentication flow for web apps uses URIs to initiate the authorization request and to return the response to the web app. The flow is similar for mobile apps. The mobile app uses an external user-agent (the device's browser) to perform authentication. Since the authorization request from a mobile app is initiated from the device's browser, you can apply sign-in principles that are similar to web apps in order to share a sign-in session between mobile apps on a device.
+
+## Session and persistent Single Sign-On
+
+Single Sign-On (SSO) allows users to authenticate once and access multiple resources without being prompted for additional credentials. Okta supports both session and persistent SSO:
+
+* **Session SSO**: Session SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications during a particular session. However, if a particular session ends, the user is prompted for their credentials again.
+* **Persistent SSO**: Persistent SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications for as long as the persistent SSO cookie is valid.
+
+The difference between persistent SSO and session SSO is that persistent SSO can be maintained across different sessions. Persistent SSO is disabled by default in Okta. To share a sign-in session with native mobile apps, you need to enable persistent SSO. 
+
+<StackSelector snippet="enablesso" />
```

Comment:
```suggestion
<StackSelector snippet="enablesso" noSelector />
```
If we have a stack selector at the top of the page, then we don't need to show one here.